recipes/pyomero-upload/meta.yaml

```diff
@@ -6,11 +6,14 @@ package:
 
 source:
   url: https://github.com/SynthSys/pyOmeroUpload/archive/v1.0.0-rc0.tar.gz
-  sha256: 80b983792b8f54caa86b9815806be9320a5eab358f84d94fef1e3ca2a42981dc
+  sha256: 9bdb948b9720bb4d0fc761be19aa9c05a062195e06e74e0339d0d4fd6e3b3dd7
+  # git_url: https://github.com/SynthSys/pyOmeroUpload.git
```

Comment:
can you please clean this up?

Reply:
Fix in commit fefb1958bc66d0a6465be7e408e26e1e1710b4c3